executor/analyze.go

```diff
@@ -238,6 +239,9 @@ func (e *AnalyzeExec) handleResultsError(ctx context.Context, concurrency int, n
 	panicCnt := 0
 	var err error
 	for panicCnt < concurrency {
+		if atomic.LoadUint32(&e.ctx.GetSessionVars().Killed) == 1 {
```

Comment:
I suggest using ```atomic.Uint32``` to avoid forgetting to use atomic.

But it is just suggestion.